.github/workflows/requirements-test.txt

```
@@ -1,5 +1,5 @@
-signac==1.8.0
-signac-flow==0.24.0
+signac==2.0.0
+signac-flow==0.25.1
```

```suggestion
signac-flow==0.25.0
```
0.25.1 doesn't exist yet: https://pypi.org/project/signac-flow/#history